index.html

```diff
@@ -0,0 +1,175 @@
+          <p class="program-card-description">Listen to the speakers from various countries about the messenges of
+            sharing and opening.</p>
+        </article>
+        <article class="program-card">
+          <div class="program-card-icon">
+            <img src="./assets/business_center_white_24dp.svg" alt="">
+          </div>
+          <h3 class="program-card-title">Business Land</h3>
+          <p class="program-card-description">In this land you will be able to explore new trends, acquire tools and
+            discuss innovative strategies to
+            expand your business.</p>
+        </article>
+        <article class="program-card">
+          <div class="program-card-icon">
+            <img src="./assets/code_white_24dp.svg" alt="">
+          </div>
+          <h3 class="program-card-title">Developer Land</h3>
+          <p class="program-card-description">In this scenario you will be able to discuss and discover the potential
+            of AI.</p>
+        </article>
+        <article class="program-card">
+          <div class="program-card-icon">
+            <img src="./assets/sports_esports_white_24dp.svg" alt="">
+          </div>
+          <h3 class="program-card-title">Gamer Land</h3>
+          <p class="program-card-description">Dive into the vast playing field this land has to offer and reveal its
+            secrets as you win a game to the
+            future.</p>
+        </article>
+        <article class="program-card">
+          <div class="program-card-icon">
+            <img src="./assets/keyboard_white_24dp.svg" alt="">
+          </div>
+          <h3 class="program-card-title">Workshops</h3>
+          <p class="program-card-description">Try creating your own work using open source license rather than just
+            watching at it.</p>
+        </article>
+      </div>
+      <a class="display display-none" href="#">SEE THE WHOLE PROGRAM</a>
+      <div class="join-button">
+        <p>Join Web Development Conference</p>
+      </div>
+    </section>
+
+    <section id="speakers" class="speakers">
+
+      <div id="flexcards" class="flex-card">
+      </div>
+
+      <div class="arrow_down">
+        <p>MORE</span></p>
+        <span><i class="fa-solid fa-angle-down fa-2x"></i>
+      </div>
+    </section>
+
+    <section class="container-fluid partner">
+      <div>
+        <h2>Partners</h2>
+        <hr>
+      </div>
+      <div>
+        <img src="./assets/HCLTech-new-logo.svg.png" alt="">
+        <img src="./assets/astrazeneca-logo-big.png" alt="">
+        <img src="./assets/2560px-Baxter.svg.png" alt="">
+        <img src="./assets/Logo_de_Steren.svg.png" alt="">
+        <img src="./assets/telcel-logo.svg" alt="">
+      </div>
+    </section>
+  </main>
+  <footer class="container-fluid">
+    <div>
+      <p>© Copyright 2023</p>
+    </div>
+    <div>
+      <p>Talent Network.</p>
+      <p>Todos los derechos reservados. </p>
+    </div>
+  </footer>
```

Comment:
- [ ] You home page on the mobile does not look as expected. The layout of the whole page on the mobile need some work to look exactly like guiding template. It has extral space to the right of the page. Kindly look into that and fix.

### Your Design
[app-mobile.webm](https://user-images.githubusercontent.com/26597432/233482830-436ac5f5-1f27-49ca-a2ff-f6ecac96b0d1.webm)

### Guiding Template
![temp-mobile-1](https://user-images.githubusercontent.com/26597432/233483264-5ead6caa-fae7-403d-b80b-b377102ab873.png)

![temp-2](https://user-images.githubusercontent.com/26597432/233483289-cf0e64a5-a814-414a-b56b-da51002fa3e9.png)

![temp-3](https://user-images.githubusercontent.com/26597432/233483314-2d4c7db5-9a6f-4af4-9f58-5cc1c439eb13.png)